apps/voting/app/src/components/VoteCard/VoteCard.js

```diff
@@ -92,28 +92,20 @@ const VoteCard = ({ vote, onOpen }) => {
           </div>
         )}
       </div>
-      <div
+      <VoteText
+        disabled
+        prefix={<span css="font-weight: bold">#{voteId}: </span>}
+        text={description || metadata}
+        title={`#${voteId}: ${description || metadata}`}
         css={`
+          overflow: hidden;
           ${textStyle('body1')};
-          /* lines per font size per line height */
-          /* shorter texts align to the top */
-          height: ${10.5 * GU}px;
+          height: ${27 * 3}px; // 27px = line-height with body1
```

Comment:
Maybe we can add a note to the comment about `3` being the number of lines we want